learntools/machine_learning/nbs/tut3.ipynb

```diff
@@ -0,0 +1,261 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "_cell_guid": "dc1e7b02-f022-48ca-a270-21f4b0c70835",
+    "_uuid": "0696f41450f17b2e0ebbbbb30a29053cde7de211"
+   },
+   "source": [
+    "**[Machine Learning Course Home Page](kaggle.com/learn/machine-learning).**\n",
+    "\n",
+    "---\n",
+    "\n",
+    "# Selecting and Data for Modeling\n",
+    "Your dataset had  too many variables to wrap your head around, or even to print out nicely.  How can you pare down this overwhelming amount of data to something you can understand?\n",
+    "\n",
+    "To show you the techniques, we'll start by picking a few variables using our intuition. Later tutorials will show you statistical techniques to  automatically prioritize variables.\n",
+    "\n",
+    "To choose variables/columns, we'll need to see a list of all columns in the dataset. That is done with the **columns** property of the DataFrame (the bottom line of code below).\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "melbourne_file_path = '../input/melbourne-housing-snapshot/melb_data.csv'\n",
+    "melbourne_data = pd.read_csv(melbourne_file_path) \n",
+    "print(melbourne_data.columns)"
```

Comment:
This is sort of a matter of preference, but if I want to display one thing from one cell, I prefer to just have it as a naked expression at the end of the cell (i.e. just `melbourne_data.columns`) rather than explicitly printing it. Usually the results are either the same or better (because if you omit the print, you can get the benefit of objects that implement rich IPython repr methods).